spec/models/guide_spec.rb

```diff
@@ -140,7 +139,7 @@
     end
 
     context 'when progress is in more than one organization' do
-      let(:another_organization) { create(:another_test_organization) }
+      let(:another_organization) { create(:another_test_organization, book: test_organization.book) }
```

Comment:
Should we add the following test here? Is it relevant?

https://github.com/mumuki/mumuki-domain/commit/5e7759a1fc27e2e1025b7f6ac5b3ddef7b354b56#diff-852e850ded5e5f33c7b8f853db8fbcd980b09b1c93978c29c67facfda1925016R90-R93

It verifies that progress is deleted only on the corresponding organization and not in others, which I think we're missing in this PR (and it's the intended behavior... right? :laughing:)

I found this on a 2019 branch back when we first started this

Reply:
Though I guess we should also update this in Laboratory then: https://github.com/mumuki/mumuki-laboratory/commit/e046bc14160f2506f60e844c2a049826da7de1d6

Reply:
> Though I guess we should also update this in Laboratory then: mumuki/mumuki-laboratory@e046bc1

that method defaults to current organization so it should be alright either way! still, it would be the responsible thing to add that :grin: